node/src/command.rs

```diff
@@ -374,14 +384,13 @@ pub fn run() -> Result<()> {
 
 				let id = ParaId::from(para_id);
 				let parachain_account =
-					AccountIdConversion::<polkadot_primitives::v2::AccountId>::into_account(&id);
+					AccountIdConversion::<polkadot_primitives::v2::AccountId>::into_account_truncating(&id);
```

Comment:
did they stop using into_account for some reason? into_account_truncating can come up with duplicate accounts at times: https://paritytech.github.io/cumulus/cumulus_primitives_core/struct.ParaId.html#method.into_account_truncating

Reply:
hmm let me look into it. I just copied the parity parachain template

Reply:
From what I can tell this is safe because we are only use it for ParaIds and pallets. never for normal addresses